### DISCUSSION TITLE
Seeking guidance on connecting Webflow with Swell

### DISCUSSION THREAD

**Discussion by jamiekim828** [swellstores/community#268] [Help] [answered]
Hi.

I am interested in connecting Swell with Webflow, however I need your guidance.
1. How can I establish connection between Webflow and Swell? If it is possible, please guide me.
2. How about using zapier? Could you please share insights on how to integrate Webflow and Swell with zapier? Are there any limitations or considerations I should be aware of?

I would greatly appreciate any advice or personal experiences regarding these topics.
Thank you in advance!

**Comment by logeshswell** [comment, marked as answer]
@jamiekim828 Swell has already integrated with Zapier and you can follow the below steps for seamless integration:
https://www.swell.is/help/integrations/zapier#zapier

**Reply by logeshswell** [reply]
Regarding webflow, what exactly do you want to achieve in webflow, so that I can assist you accordingly?

**Reply by jamiekim828** [reply]
Thank you so much for your help. Regarding Webflow, my goal is to create a online store that offers both a frontend interface and a dynamic CMS tool. I think Swell will enhance functionality of the website. I hope this integration enables me to create and manage customer loyalty programs and rewards which can significantly enhance the user experience mainly.

**Reply by logeshswell** [reply]
Wow, the use case sounds very good, but we don't have a direct integration with Webflow at the moment. I will raise a feature request to support webflow directly via Swell.
But, the immediate and fastest solution would be to integrate Swell and Webflow via Zapier, and the guide is already linked in the same thread. You could definitely achieve your use case using this method.

**Reply by jamiekim828** [reply]
Amazing. And this time I will integrate with Webflow leveraging the power of Zapier. Thank you!
